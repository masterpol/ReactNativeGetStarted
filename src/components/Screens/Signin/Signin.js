import React from 'react'
import { View, Text } from 'react-native'

const Signin = () => {
  return (
    <View>
      <Text>Signin Screen</Text>
    </View>
  )
}

export default Signin
