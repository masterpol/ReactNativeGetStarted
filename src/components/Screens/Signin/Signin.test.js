import React from 'react'
import Signin from './Signin'

describe('Signin Screen', () => {
  let props

  beforeEach(() => {
    props = {}
  })

  describe('structure', () => {
    it('should match the snapshot', () => {
      const wrapper = render(<Signin {...props} />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
