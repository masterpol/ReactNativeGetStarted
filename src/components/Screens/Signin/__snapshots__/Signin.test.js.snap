// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin Screen structure should match the snapshot 1`] = `
<View>
  <Text>
    Signin Screen
  </Text>
</View>
`;
